packages/contracts/src/dollar/core/ERC20Ubiquity.sol

```diff
@@ -73,6 +69,18 @@ abstract contract ERC20UbiquityForDiamond is
         );
     }
 
+    /// @notice getDiamond returns the diamond address
+    /// @return diamond address
+    function getDiamond() external view returns (address) {
+        return address(accessCtrl);
+    }
+
+    /// @notice setDiamond update the diamond address
+    /// @param _diamond new diamond address
+    function setDiamond(address _diamond) external onlyAdmin {
+        accessCtrl = IAccessControl(_diamond);
+    }
+
```

Comment:
Before leaving my comment I searched in the codebase to see how this method is used, excluding `t.sol` test files. It doesn't look like its being used anywhere? It simply returns an address but I don't see exactly how that address is being called or used. 

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/4975670/224260366-27af17c7-5fc9-4962-9796-aa2259842e5c.png">

Perhaps it makes more sense to abstract away the term `Diamond`, and instead just refer to it as `Manager`. 

It doesn't look like the interface is "diamond specific" (e.g. requiring "diamond" to be in the method names, but I couldn't find any implementations to confirm this as per the first part of this comment.) 

Abstracting away the term `Diamond` from our codebase grants us the flexibility to upgrade to a non diamond manager in the future and for it to still make sense.

Reply:
renamed to Manager